packages/xrpl/test/utils/getNFTokenID.test.ts

```diff
@@ -18,11 +18,4 @@ describe('getNFTokenID', function () {
       '0008125CBE4B401B2F62ED35CC67362165AA813CCA06316FFA766254000003EE'
     assert.equal(result, expectedNFTokenID)
   })
-
-  it('fails with nice error when given raw response instead of meta', function () {
```

Comment:
What's wrong with this test?

Reply:
Nothing - I was a bit hasty removing it. 

I removed the error check because I thought that there was no longer a case where `meta` could be undefined - but forgot about the case in JavaScript where this could happen. Then I saw the test fail and didn't read the title - I just thought it was checking for an error which couldn't happen anymore).